data/data-pipeline/data_pipeline/etl/sources/eamlis/etl.py

```diff
@@ -0,0 +1,63 @@
+from pathlib import Path
+import geopandas as gpd
+import pandas as pd
+from data_pipeline.config import settings
+
+from data_pipeline.etl.base import ExtractTransformLoad, ValidGeoLevel
+from data_pipeline.etl.sources.geo_utils import add_tracts_for_geometries
+from data_pipeline.utils import get_module_logger
+
+logger = get_module_logger(__name__)
+
+
+class AbandonedMineLandInventorySystem(ExtractTransformLoad):
+    """Data from Office Of Surface Mining Reclamation and Enforcement's
+    eAMLIS. These are the locations of abandoned mines.
+    """
+
+    # Metadata for the baseclass
+    NAME = "eamlis"
+    GEO_LEVEL = ValidGeoLevel.CENSUS_TRACT
+    BINARY_HAS_AMLIS: str = "Has abandoned mine"
```

Comment:
why is this here rather than in the yml file?

Reply:
Oh good catch! Thank you --- I had put it in before I wrote the YAML when I was deciding what vars I needed. THank you!

Reply:
Fixed in 702804e39047697fd3673a64fdd56256dfd82d58